# HG changeset patch
# User John W. Eaton <jwe@octave.org>
# Date 1551105046 0
#      Mon Feb 25 14:30:46 2019 +0000
# Branch stable
# Node ID 5d27954485ede62941b40d1852af15077ff210c7
# Parent  69a270a9bbfb935f3aaa1ce654bc750545ef6742
maint: Bump version for post 5.1.0 release bug fixing phase.

diff --git a/configure.ac b/configure.ac
--- a/configure.ac
+++ b/configure.ac
@@ -15,31 +15,31 @@ dnl Copyright (C) 1993-2019 John W. Eato
 ### GNU General Public License for more details.
 ###
 ### You should have received a copy of the GNU General Public License
 ### along with Octave; see the file COPYING.  If not, see
 ### <https://www.gnu.org/licenses/>.
 
 ### Initialize Autoconf
 AC_PREREQ([2.65])
-AC_INIT([GNU Octave], [5.1.0], [https://octave.org/bugs.html], [octave],
+AC_INIT([GNU Octave], [5.1.1], [https://octave.org/bugs.html], [octave],
         [https://www.gnu.org/software/octave/])
 
 ### Declare version numbers
 
 dnl Note that the version number is duplicated here and in AC_INIT because
 dnl AC_INIT requires it to be static, not computed from shell variables.
 
 ## The description of the Octave version number in the etc/HACKING.md
 ## explains how to update these numbers for release and development
 ## versions.
 
 OCTAVE_MAJOR_VERSION=5
 OCTAVE_MINOR_VERSION=1
-OCTAVE_PATCH_VERSION=0
+OCTAVE_PATCH_VERSION=1
 
 dnl PACKAGE_VERSION is set by the AC_INIT VERSION argument.
 OCTAVE_VERSION="$PACKAGE_VERSION"
 
 OCTAVE_COPYRIGHT="Copyright (C) 2019 John W. Eaton and others."
 
 OCTAVE_RELEASE_DATE="2019-02-23"
 
